Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -193,7 +193,7 @@ function New-AvsLDAPIdentitySource {
 
 <#
     .Synopsis
-     Allow customers to add an LDAPS Secure external identity source (Active Directory over LDAP) for use with single sign on to vCenter. Prefaced by Connect-SsoAdminServer
+     Recommended: Allow customers to add an LDAPS Secure external identity source (Active Directory over LDAPS) for use with single sign on to vCenter.
```

Comment:
This is confusing. Recommend ... Secure. But this one is not adding a secure/LDAPS source, is it?

Reply:
Yes, this was a mistake. Good catch

Reply:
Yeah, I'm not sure the best way to write this. But this is the one for adding a secure LDAPS source. I just remember feedback that they wanted to have it more explicitly stated that we really want customers to run this command and not the LDAP one

Reply:
To align to other Microsoft docs/terminology & ldap cmdlet synopsis, I suggest: "Recommended: Allow customers to add a secure external identity source (Active Directory over LDAPS) for use with vCenter Single Sign-On."

Reply:
Right, sorry, I was thinking I'm commenting on an unsecure version...